### ISSUE TITLE
Unable to run groc on Ubuntu 13.04 64 bit 

### ISSUE THREAD

**Issue by aviralg** [nevir/groc#147]
I untared node binary in /usr folder. 

`sudo npm install -g groc`  worked fine.

when i call groc from terminal, i get the following error -

module.js:340
    throw err;
          ^
Error: Cannot find module './lib/cli'
  at Function.Module._resolveFilename (module.js:338:15)
  at Function.Module._load (module.js:280:25)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (/usr/local/lib/node_modules/groc/index.js:5:17)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Module.require (module.js:364:17)
  at require (module.js:380:17)
  at Object.<anonymous> (/usr/local/lib/node_modules/groc/bin/groc:3:1)
  at Module._compile (module.js:456:26)
  at Object.Module._extensions..js (module.js:474:10)
  at Module.load (module.js:356:32)
  at Function.Module._load (module.js:312:12)
  at Function.Module.runMain (module.js:497:10)
  at startup (node.js:119:16)
  at node.js:902:3

**Comment by kellynichols** [comment]
I'm having this same problem on OS X 10.9.1. Could it be that the coffee files are not compiled?

**Comment by johnlouis-swaine-axomic** [comment]
Same problem on OS X 10.9.1

**Comment by kellynichols** [comment]
I was able to get around this by manually compiling the coffee files in /lib, /lib/styles, and /lib/utils and then fixing the jade doctype issue that others have mentioned. Clearly not a long term fix, but it works for the moment.

**Comment by cspotcode** [comment]
This is caused by a change to CoffeeScript's behavior regarding the ability to `require()` `.coffee` files without compiling them ahead of time.  See [the changelog](http://coffeescript.org/#changelog) for CoffeeScript 1.7.0:

> When requiring CoffeeScript files in Node you must now explicitly register the compiler. This can be done with `require 'coffee-script/register'` or `CoffeeScript.register()`. Also for configuration such as Mocha's, use `coffee-script/register`.

groc doesn't do this; it merely `require`s CoffeeScript.
https://github.com/nevir/groc/blob/master/index.js#L1

**Comment by variousauthors** [comment]
I'm having this same issue. Any progress?

Can we start with details about @kellynichols' fix? I'm not a coffee-script guy, how should we manually compile these files? I tries installing `coffee-script` via npm and doing

`coffee -c lib/`

I received the following error:

```
/usr/lib/node_modules/groc/lib/styles/default/behavior.coffee:1:19: error: unexpected <
tableOfContents = <%= JSON.stringify(tableOfContents) %>
```

(using docco-husky for now, until I can use groc)

**Comment by cspotcode** [comment]
@variousauthors You can use the fix in pull request #155.  It's a one-line change and avoids the need to manually compile anything.

EDIT: Specifically, this one-line change:
https://github.com/cspotcode/groc/commit/bf6178d427aaaf9da44d53fe8b0a7b7553939d1a
